tests/contrib/gp/test_models.py

```diff
@@ -209,16 +212,16 @@ def test_inference_svgp():
     X = dist.Uniform(torch.zeros(N), torch.ones(N)*5).sample()
     y = 0.5 * torch.sin(3*X) + dist.Normal(torch.zeros(N), torch.ones(N)*0.5).sample()
     kernel = RBF(input_dim=1)
-    Xu = torch.linspace(0, 5, 10)
+    Xu = torch.arange(0, 5.5, 0.5)
 
     svgp = SparseVariationalGP(X, y, kernel, Xu, Gaussian())
-    svgp.optimize(optim.Adam({"lr": 0.01}), num_steps=1000)
+    svgp.optimize(optim.Adam({"lr": 0.01}), num_steps=2000)
 
-    Xnew = torch.linspace(0, 5, 100)
+    Xnew = torch.arange(0, 5.05, 0.05)
```

Comment:
I find `linspace` easier to reason about than `arange`. Is `linspace` not supported in CUDA?

Reply:
Yes, this will throw an error to that effect.